src/BrightScriptDebugSession.ts

```diff
@@ -989,6 +1000,59 @@ export class BrightScriptDebugSession extends DebugSession {
         await Promise.all(promises);
     }
 
+    /**
+     * Will search the project files for the comment "\' vs_code_tracker_entry" and replace it with the code needed to start the TrackerTask.
+     * @param stagingPath
+     */
+    public async injectTrackerTaskCode(stagingPath: string) {
+        // Search for the tracker task entry injection point
+        let trackerEntryTerm = `('\\s*vs_code_tracker_entry[^\\S\\r\\n]*)`;
```

Comment:
Let's make `vs_code` one word `vscode`. 

We should probably use the world `rale` somewhere in this key? `'vscode_rale_tracker_entry`. Thoughts?